meson.build

```diff
@@ -1,15 +1,26 @@
 project(
     'vs-fltk',
     ['c', 'cpp'],
-    #<inject-version>
-    version: '0.1.2-alpha',
-    #</inject-version>
+    version: files('version.txt'),
     meson_version: '>= 1.1',
     default_options: [
         'c_std=gnu23',
         'cpp_std=gnu++23',
     ],
 )
+
+version_components = meson.project_version().split('.')
```

Comment:
Now I see all this stuff I added to meson is already being done in index.ts. Should I just remove this from meson?